styles/_buttons.scss

```diff
@@ -26,13 +26,15 @@
   &:hover,
   &:active,
   &:focus,
-  &.button-primary {
+  &.button-primary,
+  &[aria-pressed="true"] {
```

Comment:
Could we move this to where it is needed only?
I'm thinking that the vote buttons which only show the svg carets might not need all this other styles (?)